Userland/Libraries/LibWeb/Painting/Paintable.h

```diff
@@ -53,10 +53,13 @@ class Paintable : public RefCounted<Paintable> {
     virtual HitTestResult hit_test(Gfx::IntPoint const&, HitTestType) const;
 
     virtual bool wants_mouse_events() const { return false; }
-    virtual void handle_mousedown(Badge<EventHandler>, const Gfx::IntPoint&, unsigned button, unsigned modifiers);
-    virtual void handle_mouseup(Badge<EventHandler>, const Gfx::IntPoint&, unsigned button, unsigned modifiers);
-    virtual void handle_mousemove(Badge<EventHandler>, const Gfx::IntPoint&, unsigned buttons, unsigned modifiers);
+    // When these methods return true, the DOM event with the same name will be
+    // fired at the layout node's associated DOM node.
+    virtual bool handle_mousedown(Badge<EventHandler>, const Gfx::IntPoint&, unsigned button, unsigned modifiers);
+    virtual bool handle_mouseup(Badge<EventHandler>, const Gfx::IntPoint&, unsigned button, unsigned modifiers);
+    virtual bool handle_mousemove(Badge<EventHandler>, const Gfx::IntPoint&, unsigned buttons, unsigned modifiers);
```

Comment:
Could we return an enum (e.g. DispatchEventOfSameName::{No,Yes}) instead of a bool? Therefore it's obvious what the return value means without having to look for the comment.